prov/efa/src/rdm/rxr_env.c

```diff
@@ -83,14 +82,21 @@ void rxr_env_param_get(void)
 		abort();
 	};
 
+	if (getenv("FI_EFA_MTU_SIZE")) {
+		fprintf(stderr,
+			"FI_EFA_MTU_SIZE env variable detected! The use of this variable has been deprecated and as such execution cannot proceed.\n");
+		abort();
+	};
+
 	fi_param_get_int(&efa_prov, "tx_min_credits", &rxr_env.tx_min_credits);
 	if (rxr_env.tx_min_credits <= 0) {
-		EFA_WARN(FI_LOG_EP_DATA,
-			"FI_EFA_TX_MIN_CREDITS was set to %d, which is <= 0."
-			"This value will cause EFA communication to deadlock."
-			"To avoid that, the variable was reset to %d\n",
-			rxr_env.tx_min_credits, RXR_DEF_MIN_TX_CREDITS);
-		rxr_env.tx_min_credits = RXR_DEF_MIN_TX_CREDITS;
+		fprintf(stderr,
+			"FI_EFA_TX_MIN_CREDITS was set to %d, which is <= 0.\n"
+			"This value will cause EFA communication to deadlock.\n"
+			"Please unset the environment variable or set it to a positive number\n"
+			"You application will now abort.",
```

Comment:
```suggestion
			"Please unset the environment variable or set it to a positive number.\n"
			"Your application will now abort.",
```

No strong preference on the punctuation, but it should be consistent.